CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This file documents all notable changes to the GEOS-Chem repository since versio
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased [14.0.2]
+### Changed
+- Rewrote code to avoid memory leaks (identified by the code sanitizer)
```

Comment:
I think the updates are more apt under Fixes: "Fixed memory leaks identified by code sanitizer"

Reply:
Good point @lizziel.  I'll update that.

Reply:
Now changed in commit 40794df15b351db399c5225fb5c90031ae9ab3d4